cardano-db-sync/src/Cardano/DbSync/Plugin/Default/Insert.hs

```diff
@@ -189,18 +189,19 @@ insertTxOut _tracer txId index txout =
 
 insertTxIn
     :: MonadIO m
-    => Trace IO Text -> DB.TxId -> Ledger.TxIn
+    => Trace IO Text -> DB.TxId -> Word32 -> Ledger.TxIn
```

Comment:
To match the schema, that should be `Word16 .

Reply:
Matched according to https://github.com/input-output-hk/cardano-db-sync/blob/5296767b8a2c6f3bd515bf10c4474f7d13b12430/cardano-db-sync/src/Cardano/DbSync/Plugin/Default/Insert.hs#L178

Reply:
Yes, but the schema says `Word16` : https://github.com/input-output-hk/cardano-db-sync/blob/536d485abe9a290d04c0ac7649b05efb2ec825d2/cardano-db/src/Cardano/Db/Schema.hs#L95

Reply:
Nevermind one is `Word32` and one is `Word16`.